src/client/pages/EventPage/components/AppsOverTimeStatistics.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { VictoryChart, VictoryAxis, VictoryLine, VictoryTheme } from 'victory';
+
+import EventStatisticsComponent from './EventStatisticsComponent';
+
+export default class AppsOverTimeStatistics extends EventStatisticsComponent {
+  render() {
+    const { statistics } = this.props;
+
+    // Create the data array needed to make the line chart
+    const appsTimeData: { date: string; appCount: number }[] = [];
+
+    Object.keys(statistics.appsOverTime).forEach(key => {
+      if (key !== 'null') {
```

Comment:
Do we know where these `null` values are coming? It seems odd that we're screening for null dates in the front-end code.

Reply:
This seems to only appear in the local development web app. After running `npm run migrate`, the event that appears populated has 1 mock applicant with a timestamp of null. Are we certain that such edge case won't occur in production? If so, I can of course remove the check.

Reply:
The database mocks are sorely out of date and buggy. I can give you a backup of the production database to use as your development database if you would prefer to use those. Timestamps should never be null

Reply:
That would be great! Although I'm a bit concerned about set up since Panda and I had quite some trouble making my local repo work when I cloned it for the first time. I'm not using docker atm either, instead using a workaround Lisa told me about :P

Reply:
I use a Mongo database provided by mongodb.com since the Docker mongo lags my laptop if I choose to dev on there.

Reply:
Can I DM you on Slack later about this?

Reply:
Totally yeah